scripts/get

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
```

Comment:
Does mac os have `/usr/bin/env`?

Reply:
'I have no idea' to both. Let's merge for now, open a follow-up issue, conscript Daniel?